backend/app/widget/project.py

```diff
@@ -252,10 +252,10 @@ def build(self):
             Returns the status code returned from the build
         """
         rep = MQReporter(self.app, self.task_id)
-        rep.console(["gprbuild", "-q", "-P", self.gpr.get_name(), "-gnatwa"])
+        rep.console(["gprbuild", "-q", "-P", self.gpr.get_name(), "-gnatwa", "-gnata"])
 
         main_path = os.path.join(self.remote_tempd, self.gpr.get_name())
-        line = ["gprbuild", "-q", "-P", main_path, "-gnatwa"]
+        line = ["gprbuild", "-q", "-P", main_path, "-gnatwa", "-gnata"]
```

Comment:
We don't want to add -gnata here. We should add it to the project file. We should probably add -gnatwa to the project file too to simplify this build line so that users can easily replicate similar behavior when they download the example. Can you move these two switches to the Compiler package in the project file: https://github.com/gusthoff/ada-learning-material/blob/intro_to_ada/review/contracts/20200417_pragmas/backend/app/widget/static/templates/inline_code/main.gpr

Reply:
Thanks! I was actually looking for that file...